server.cpp

```diff
@@ -77,17 +76,6 @@ void connection::do_read() {
 void connection::do_write() {
 	auto self(shared_from_this());
 
-	/*reply_.status = reply::ok;
-	header head0;
-	head0.name = "Content-Length";
-	head0.value = std::to_string(reply_.content.size());
-	reply_.headers.push_back(head0);
-        
-	header head1;
-	head1.name = "Content-Type";
-	head1.value = "text/plain";
-	reply_.headers.push_back(head1);*/
-
```

Comment:
Good catch! As mentioned in lecture, having commented code is unnecessary and since we have version control, there is no need to have anything like this.